class-wp-bootstrap-navwalker.php

```diff
@@ -212,12 +212,12 @@ public function start_el( &$output, $item, $depth = 0, $args = null, $id = 0 ) {
 
 			// If the item has_children add atts to <a>.
 			if ( $this->has_children && 0 === $depth ) {
-				$atts['href']          = '#';
-				$atts['data-toggle']   = 'dropdown';
-				$atts['aria-haspopup'] = 'true';
-				$atts['aria-expanded'] = 'false';
-				$atts['class']         = 'dropdown-toggle nav-link';
-				$atts['id']            = 'menu-item-dropdown-' . $item->ID;
+				$atts['href']          	= '#';
+				$atts['data-bs-toggle']	= 'dropdown';
```

Comment:
Welcome to WP Bootstrap Navwalker and thanks for contributing.

One note to your propsed changes: Switching from `$atts['data-toggle' = 'dropdown'` to  `$atts['data-bs-toggle' = 'dropdown'` will break the walker for people using Bootstrap 4.